dev-docs/pb7-notes.md

```diff
@@ -0,0 +1,79 @@
+
+Also, Bid Adapters (not all adapters) no longer have access to storage manager unless explicitly allowed by the publisher. We believe bidding functionality should rarely if ever need this access and that this extra functionality included in bid adapters must be consented to by the installer in their configuration. The following bid adapters' unit tests were affected by this change: Adagio, Adnuntius, AP Stream, Concert, Conversant, Craft, Criteo, E-Planning, Invibes, Kargo, Quantcast, Trion, Unicorn, and Vidazoo. Adapters simply setting a random identifier in the first party local storage or cookie should consider if the popular shared id user id submodule can fully achieve this functionality for them. See storageAllowed in the [Publisher API Reference]({{site.baseurl}}/dev-docs/publisher-api-reference.html) 
+
+Adapters are also now not allowed to return alternate bidder codes unless allowed by the publisher. See allowAlternateBidderCodes in the [Publisher API Reference]({{site.baseurl}}/dev-docs/publisher-api-reference.html).
+
+Finally, adapters known to use http1 bidding endpoints may now have notices appended to their documentation. 
+
+## Regulatory and consent strings
+
+TCF1 is no longer supported by the Consent Management (GDPR) module. The Consent Management USP module now defaults to framework 'iab' and will work without configuration if installed and a functioning `__uspapi()` exists on the page.
+
+## First Party Data
+
+Support has been removed for `setConfig('fpd')`, `config.getLegacyFpd`, `config.convertAdUnitFpd` and related relics of legacy first party data. Publishers should use the methods described in [First Party Data]({{site.baseurl}}/features/firstPartyData.html).
+
+First party data can also now be set on a specific auction. This is useful for example on infinite scroll pages when contextual segments change, or when the publisher wishes to express the context of an instream video ad but not the display advertising. As part of this change the ortb2 object is now made a part of bid requests, instead of necesitating a getConfig call by an adapter. This has led to slight changes in many dozens of bid adapters and substantial changes to all of the RTD adapters. 
+
+## Prebid Server Adapter
+
+An undocumented feature "Stored Auction Response" has been deprecated. 
+
+The Prebid Server committee moved the ortb2 location of bidder parameters from `imp[].ext.BIDDER` to `imp[].ext.prebid.bidder.BIDDER`. PBS versions before [insert version number here], released on [insert release date or year here], are not compatible with Prebid 7+.
+
+## Build changes
+
+In later 6.x versions, an improvement in the build process introduced some undesirable behavior for users loading the library twice on the same pageview to the same global library name. If you load Prebid.js twice on a page using the same global, the second load is now prevented. If your goal is to load a module late or on demand, it is possible to load it in isolation.
```

Comment:
@dgirardi could you review this statement for accuracy

Reply:
I have two reservations:
- I wouldn't want to give the impression that multiple loads are fine up until a particular version - we don't know that, and if it's true it's by accident.
- while it's technically true that you can load single modules separately, we never tested that either and it's likely to be harder than it sounds, at least for some modules.

I would suggest something along the lines of 

> Including multiple conflicting copies of Prebid in the same page now has no effect - after the first initialization, any other instance of Prebid that shares the same global variable name will not run. Loading Prebid multiple times is known to cause problems since version 6.17, and its behavior was undefined before then.